uk.co.compucorp.civicrm.tasksassignments/CRM/Tasksassignments/Reminder.php

```diff
@@ -1009,4 +993,77 @@ private static function getUfMatchContact($contact_id) {
     return $uf_match_data;
   }
 
+  /**
+   * Constructs a query condition that excludes "completed" activities.
+   *
+   * @return string
+   */
+  private static function getCompletedActivitiesExclusionQuery() {
+    $documentIncompleteStatuses = implode(',', self::_getDocumentIncompleteStatuses());
+    $documentTypesIds = implode(',', self::_getTypesIdsForComponent('CiviDocument'));
+    $taskIncompleteStatuses = implode(',', self::_getTaskIncompleteStatuses());
+    $taskTypesIds = implode(',', self::_getTypesIdsForComponent('CiviTask'));
+
+    return "(
+      (
+        a.status_id IN ($documentIncompleteStatuses)
+        AND a.activity_type_id IN ($documentTypesIds)
+      )
+      OR
+      (
+        a.status_id IN ($taskIncompleteStatuses)
+        AND a.activity_type_id IN ($taskTypesIds)
+      )
+    )";
+  }
+
+  /**
+   * Returns incomplete statuses for Documents
+   *
+   * @return array
+   */
+  private static function _getDocumentIncompleteStatuses() {
+    return [
+      CRM_Tasksassignments_BAO_Document::STATUS_AWAITING_UPLOAD,
+      CRM_Tasksassignments_BAO_Document::STATUS_AWAITING_APPROVAL
+    ];
+  }
+
+  /**
+   * Returns Activity Types IDs for a given component name,
+   * for example, "CiviTask" or "CiviDocument"
+   *
+   * @param string $componentName eg
+   * @return array
+   */
+  private static function _getTypesIdsForComponent($componentName) {
```

Comment:
No need to start the function name with an underscore. Making it private is enough.

Reply:
Done. As discussed, I removed all underscores from all private functions. I performed:
1) search by "function _" (ensured the functions are private)
2) replaced all appearances of "_functionName" with "functionName" in the file